metadata-ingestion/pyproject.toml

```diff
@@ -38,10 +40,18 @@ extend-ignore = [
     "E203",  # Ignore whitespace before ':' (matches Black)
     "B019",  # Allow usages of functools.lru_cache
     "B008",  # Allow function call in argument defaults
+    "COM812", # Avoid conflicts with formatter
+    "C400",  # Ignore Unnecessary generator
+    "C410",  # Ignore Unnecessary list literal passed to `list()`
+    "C414",  # Ignore Unnecessary `reversed()` call within `sorted()`
+    "C417",  # Ignore Unnecessary  `map()` usage (rewrite using a list comprehension)
+    "C419",  # Ignore Unnecessary list comprehension
```

Comment:
seems like all of these should also be in the "enable later" section

we should also explain why each "enable later" rule was not enabled now